### ISSUE TITLE
Window appeared saying 'Error: Settings will be reset.' Since then issues.

### ISSUE THREAD

**Issue by Innitinnuit** [OpenShot/openshot-qt#3891]
**Describe the bug:**
I tried to open OPS as normal and an error message window appeared with the following message. Now project files either briefly load into the timeline and then crash when play is clicked. Or crash as soon as the file tries to open. 

**""Error loading settings file: /blah/.openshot_qt/openshot.settings. Settings will be reset."**

**Steps to reproduce the behavior:**
Unknown as all I did was try to open OPS by clicking as usual.

**Expected behavior:**
For project files to load and work.

**System Details:**
- OPS Version 2.5.1 dev20.2.5-dev2 libpenshot
- Ubuntu 20.04.1 LTS20.04

**Log Files:**
I have included debug logfiles and ordinary ones.

`5:35:28 INFO app: ------------------------------------------------15:35:28 INFO app: Wed Dec 2 15:35:28 202015:35:28 INFO app: Starting new session15:35:28 INFO app: ------------------------------------------------15:35:28 INFO app: OpenShot (version 2.5.1-dev2)15:35:28 INFO app: ------------------------------------------------15:35:28 INFO app: openshot-qt version: 2.5.1-dev215:35:28 INFO app: libopenshot version: 0.2.5-dev215:35:28 INFO app: platform: Linux-5.4.0-56-generic-x86_64-with-glibc2.2915:35:28 INFO app: processor: x86_6415:35:28 INFO app: machine: x86_6415:35:28 INFO app: python version: 3.8.515:35:28 INFO app: qt5 version: 5.12.815:35:28 INFO app: pyqt5 version: 5.14.115:35:28 INFO language: Qt Detected Languages: ['en-GB', 'en']15:35:28 INFO language: LANG Environment Variable: en_GB.UTF-815:35:28 INFO language: LOCALE Environment Variable:15:35:28 INFO language: OpenShot Preference Language: Default15:35:28 INFO project_data: Setting default profile to HD 720p 30 fps15:35:28 INFO app: Setting font to /usr/lib/python3/dist-packages/openshot_qt/images/fonts/Ubuntu-R.ttf15:35:28 INFO logger_libopenshot: Connecting to libopenshot with debug port: 555615:35:28 INFO app: Setting custom dark theme15:35:29 INFO ui_util: Initializing UI for MainWindow15:35:29 INFO webengine: WebEngine backend initializing15:35:29 INFO thumbnail: Starting thumbnail server listening on port 5188715:35:29 INFO transition_model: updating transitions model.15:35:29 INFO effects_model: updating effects model.15:35:29 INFO version: Found current version: {"openshot_version": "2.5.1"}15:35:29 INFO emoji_model: updating emoji model.15:35:30 INFO main_window: InitCacheSettings15:35:30 INFO main_window: cache-mode: CacheMemory15:35:30 INFO main_window: cache-limit-mb: 25015:35:30 INFO main_window: Creating CacheMemory object with 262144000 byte limit15:35:30 INFO preview_thread: QThread Start Method Invoked15:35:30 INFO main_window: Clear all thumbnails: /home/navin/.openshot_qt/thumbnail15:35:30 INFO main_window: Clear all animations: /home/navin/.openshot_qt/blender15:35:30 INFO main_window: Clear all titles: /home/navin/.openshot_qt/title15:35:30 INFO main_window: updateStatusChanged15:35:30 INFO main_window: updateStatusChanged15:35:30 INFO app: Process command-line arguments: ['/usr/bin/openshot-qt']15:35:30 INFO main_window: recover_backup15:35:30 INFO project_data: Setting default profile to HD 720p 30 fps15:35:30 INFO preview_thread: player Position(): 115:35:30 INFO video_widget: Load: Set video widget display aspect ratio to: 1.777777791023254415:35:30 INFO video_widget: Set video widget pixel aspect ratio to: 1.015:35:30 INFO main_window: updateStatusChanged15:35:30 INFO main_window: foundCurrentVersion: Found the latest version: 2.5.115:35:31 INFO preview_thread: player Position(): 115:35:31 INFO webengine: Registering WebChannel connection with WebEngine15:35:31 INFO timeline: Adjusting max size of preview image: PyQt5.QtCore.QSize(718, 404)15:35:31 INFO preview_thread: player Position(): 115:35:31 ERROR webengine: angular-animate.min.js@L14: Uncaught TypeError: m.module(...).info is not a function15:35:31 INFO webview: Qt Ready15:35:31 INFO webview: Angular Ready15:35:44 INFO main_window: Clear all thumbnails: /home/navin/.openshot_qt/thumbnail15:35:44 INFO main_window: Clear all animations: /home/navin/.openshot_qt/blender15:35:44 INFO main_window: Clear all titles: /home/navin/.openshot_qt/title15:35:44 INFO project_data: Setting default profile to HD 720p 30 fps15:35:44 INFO project_data: Loading project file: /home/navin/Documents/ARG/OpenSHot Files/Untitled Project1.osp15:35:44 INFO assets: Using existing asset folder /home/navin/Documents/ARG/OpenSHot Files/Untitled Project1_assets15:35:44 INFO assets: Using existing asset folder /home/navin/Documents/ARG/OpenSHot Files/Untitled Project1_assets15:35:44 INFO assets: Using existing asset folder /home/navin/Documents/ARG/OpenSHot Files/Untitled Project1_assets15:35:44 INFO project_data: checking project files...15:35:44 INFO project_data: checking file /home/navin/Desktop/rvgtrailer5.mp415:35:44 INFO project_data: checking file /home/navin/Desktop/untitled1.mp315:35:44 INFO project_data: checking file /home/navin/Documents/ARG/EPISODES/13/Untitled Project1.mp315:35:44 INFO project_data: checking file /home/navin/Desktop/untitled.mp315:35:44 INFO project_data: checking file /home/navin/Downloads/boy-2328038_640.png15:35:44 INFO project_data: checking file /home/navin/Downloads/On-air-Thinkstocck.jpg15:35:44 INFO project_data: checking file /home/navin/Downloads/american-christmas-traditions-gettyimages-487756624.jpg15:35:44 INFO project_data: checking file /home/navin/Downloads/oekaki-2009817_640.jpg15:35:44 INFO project_data: checking file /home/navin/Documents/ARG/EPISODES/13/title.png15:35:44 INFO project_data: checking file /home/navin/Documents/ARG/EPISODES/13/DISWELL.png15:35:44 INFO project_data: checking file /home/navin/Downloads/index999999999999.jpeg15:35:44 INFO project_data: checking file /home/navin/Downloads/Waipod Phetsuphan , Ding Ding Dong.mp4.mp315:35:44 INFO project_data: checking file /home/navin/Downloads/zen-garden.mp315:35:44 INFO project_data: checking file /home/navin/Downloads/underglow.mp315:35:44 INFO project_data: checking file /home/navin/Documents/Projects/YouTube/RVG/Every Episode/deleted scenes.mp415:35:44 INFO project_data: checking file /home/navin/Documents/Projects/YouTube/RVG/Every Episode/y2mate.com - vietnam_traditional_folk_music_yvoJdhkxYIs_160kbps.webm15:35:44 INFO project_data: checking file /home/navin/Documents/Projects/YouTube/RVG/Misc/endtitle.png15:35:44 INFO project_data: checking file /home/navin/Desktop/90sweb.png15:35:44 INFO project_data: checking file /home/navin/Downloads/metaphor-1209691_1920.jpg15:35:44 INFO project_data: checking file /home/navin/Downloads/33722ca883044584c5ce7e1a8ee96550.webp15:35:44 INFO project_data: checking file /home/navin/Downloads/error-2129569_1280.jpg15:35:44 INFO project_data: checking file /home/navin/Desktop/trailerwallpaper.jpg15:35:44 INFO project_data: checking file /home/navin/Desktop/bloopers.png15:35:44 INFO project_data: checking file /home/navin/Downloads/179223__efecto-fundador__digital-error.wav15:35:44 INFO project_data: checking file /home/navin/Desktop/kradenoe_solntse_1943_512kb.mp415:35:44 INFO project_data: checking file /home/navin/Desktop/pagan-cross.mp315:35:44 INFO project_data: checking file /home/navin/Downloads/al-andalus.mp315:35:44 INFO project_data: checking file /home/navin/Desktop/personalanswer.png15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME10-RVG.png15:35:44 INFO project_data: checking file /home/navin/Downloads/pagan-cross.mp315:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME9-RVG.png15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME8-RVG.png15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME7-RVG.png15:35:44 INFO project_data: checking file /home/navin/Downloads/dire-space-emergency.mp315:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME6-RVG.png15:35:44 INFO project_data: checking file /home/navin/Downloads/love-3556651_640.png15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME5-RVG.png15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME4-RVG.png15:35:44 INFO project_data: checking file /home/navin/Downloads/signs-to-nowhere.mp315:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME3-RVG.png15:35:44 INFO project_data: checking file /home/navin/Downloads/index99.jpeg15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME2-RVG.png15:35:44 INFO project_data: checking file /home/navin/Desktop/fix.png15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME-RVG.png15:35:44 INFO project_data: checking file /home/navin/Downloads/subtitle video.mp415:35:44 INFO project_data: checking file /home/navin/Desktop/wall-1217083_1280(1).jpg15:35:44 INFO project_data: checking file /home/navin/Downloads/index.jpeg15:35:44 INFO project_data: checking file /home/navin/Downloads/insfdfsdfdex.jpeg15:35:44 INFO project_data: checking file /home/navin/Downloads/Lilac-Bowls-335ml-Plastic-LAVE2BOWL_th2.jpg15:35:44 INFO project_data: checking file /home/navin/Downloads/cheesy-ranch-burger.jpg15:35:44 INFO project_data: checking file /home/navin/Downloads/images.jpeg15:35:44 INFO project_data: checking file /home/navin/Downloads/indekkkx.jpeg15:35:44 INFO project_data: checking file /home/navin/Downloads/index890.jpeg15:35:44 INFO project_data: checking file /home/navin/Downloads/index98.jpeg15:35:44 INFO project_data: checking file /home/navin/Downloads/index9.jpeg15:35:44 INFO project_data: checking file /home/navin/Downloads/710tVLiJOsL.jpg15:35:44 INFO project_data: checking file /home/navin/Downloads/HLB1c9ehP9rqK1RjSZK9q6xyypXaD.jpg15:35:44 INFO project_data: checking file /home/navin/Downloads/food-2202399_640.jpg15:35:44 INFO project_data: checking file /home/navin/Downloads/boy-179376_640.jpg15:35:44 INFO project_data: checking file /home/navin/Downloads/worm-1439069_640.jpg15:35:44 INFO project_data: checking file /home/navin/Downloads/ca63640d6a175b2b259ad0396b094c19.jpg15:35:44 INFO project_data: checking file /home/navin/Downloads/napkins-5073819_640.jpg15:35:44 INFO project_data: checking file /home/navin/Downloads/il_340x270.774243765_5c7h.jpg15:35:44 INFO project_data: checking file /home/navin/Downloads/fancy-a-spit-roast-card.jpg15:35:44 INFO project_data: checking file /home/navin/Downloads/Filthy-Animal-Greeting-Card.webp15:35:44 INFO project_data: checking file /home/navin/Desktop/Rvg-backgroundmusic-trailer.wav15:35:44 INFO project_data: checking file /home/navin/Desktop/rvgtrailer4.mp415:35:44 INFO project_data: checking file /home/navin/Downloads/rvg 13.mp415:35:44 INFO project_data: Project data: openshot 2.5.1-dev2, libopenshot 0.2.5-dev215:35:45 INFO preview_thread: player Position(): 115:35:46 INFO video_widget: Load: Set video widget display aspect ratio to: 1.777777791023254415:35:46 INFO video_widget: Set video widget pixel aspect ratio to: 1.015:35:46 INFO main_window: updateStatusChanged15:35:46 INFO main_window: updateStatusChanged15:35:46 INFO main_window: updateStatusChanged15:35:46 INFO main_window: updateStatusChanged15:35:46 INFO preview_thread: player Position(): 115:35:46 INFO main_window: Loaded project /home/navin/Documents/ARG/OpenSHot Files/Untitled Project1.osp15:35:51 INFO preview_thread: previewFrame: 458, player Position(): 458`

**THE OTHER LOG FILE FOR DEBUGGING MODE**

` Unhandled Exception: Stack Trace ----/lib/x86_64-linux-gnu/libopenshot.so.19 openshot::FFmpegReader::Close() 0x62c [0x7f84f813091c]/lib/x86_64-linux-gnu/libopenshot.so.19 openshot::FrameMapper::Close() 0x269 [0x7f84f8156989]/lib/x86_64-linux-gnu/libopenshot.so.19 openshot::Clip::Close() 0x26e [0x7f84f810f23e]/lib/x86_64-linux-gnu/libopenshot.so.19 openshot::Timeline::update_open_clips(openshot::Clip*, bool) 0x6b5 [0x7f84f816ccc5]/lib/x86_64-linux-gnu/libopenshot.so.19 openshot::Timeline::find_intersecting_clips(long, int, bool) 0x36a [0x7f84f816d79a]/lib/x86_64-linux-gnu/libopenshot.so.19 openshot::Timeline::GetFrame(long) 0x7b1 [0x7f84f816e151]/lib/x86_64-linux-gnu/libopenshot.so.19 openshot::PlayerPrivate::getFrame() 0x7f [0x7f84f8195fff]/lib/x86_64-linux-gnu/libopenshot.so.19 openshot::PlayerPrivate::run() 0xa1 [0x7f84f8196121]/lib/x86_64-linux-gnu/libopenshot-audio.so.7 juce::Thread::threadEntryPoint() 0xd9 [0x7f84e513eb09]/lib/x86_64-linux-gnu/libopenshot-audio.so.7 0x136c3d [0x7f84e513ec3d]/lib/x86_64-linux-gnu/libpthread.so.0 0x9609 [0x7f851780c609]/lib/x86_64-linux-gnu/libc.so.6 clone 0x43 [0x7f8517948293]---- End of Stack Trace -------- Unhandled Exception: Stack Trace ----`

**HERE ARE THE ORIGINAL ERROR LOGS**

`15:35:28 INFO app: ------------------------------------------------ 15:35:28 INFO app: Wed Dec 2 15:35:28 202015:35:28 INFO app: Starting new session15:35:28 INFO app: ------------------------------------------------ 15:35:28 INFO app: OpenShot (version 2.5.1-dev2)15:35:28 INFO app: ------------------------------------------------ 15:35:28 INFO app: openshot-qt version: 2.5.1-dev2 15:35:28 INFO app: libopenshot version: 0.2.5-dev2 15:35:28 INFO app: platform: Linux-5.4.0-56-generic-x86_64-with-glibc2.29 15:35:28 INFO app: processor: x86_64 15:35:28 INFO app: machine: x86_64 15:35:28 INFO app: python version: 3.8.5 15:35:28 INFO app: qt5 version: 5.12.8 15:35:28 INFO app: pyqt5 version: 5.14.1 15:35:28 INFO language: Qt Detected Languages: ['en-GB', 'en'] 15:35:28 INFO language: LANG Environment Variable: en_GB.UTF-8 15:35:28 INFO language: LOCALE Environment Variable: 15:35:28 INFO language: OpenShot Preference Language: Default 15:35:28 INFO project_data: Setting default profile to HD 720p 30 fps 15:35:28 INFO app: Setting font to /usr/lib/python3/dist-packages/openshot_qt/images/fonts/Ubuntu-R.ttf 15:35:28 INFO logger_libopenshot: Connecting to libopenshot with debug port: 5556 15:35:28 INFO app: Setting custom dark theme 15:35:29 INFO ui_util: Initializing UI for MainWindow 15:35:29 INFO webengine: WebEngine backend initializing 15:35:29 INFO thumbnail: Starting thumbnail server listening on port 51887 15:35:29 INFO transition_model: updating transitions model. 15:35:29 INFO effects_model: updating effects model. 15:35:29 INFO version: Found current version: {"openshot_version": "2.5.1"} 15:35:29 INFO emoji_model: updating emoji model. 15:35:30 INFO main_window: InitCacheSettings 15:35:30 INFO main_window: cache-mode: CacheMemory 15:35:30 INFO main_window: cache-limit-mb: 250 15:35:30 INFO main_window: Creating CacheMemory object with 262144000 byte limit 15:35:30 INFO preview_thread: QThread Start Method Invoked 15:35:30 INFO main_window: Clear all thumbnails: /home/navin/.openshot_qt/thumbnail 15:35:30 INFO main_window: Clear all animations: /home/navin/.openshot_qt/blender 15:35:30 INFO main_window: Clear all titles: /home/navin/.openshot_qt/title 15:35:30 INFO main_window: updateStatusChanged 15:35:30 INFO main_window: updateStatusChanged 15:35:30 INFO app: Process command-line arguments: ['/usr/bin/openshot-qt'] 15:35:30 INFO main_window: recover_backup 15:35:30 INFO project_data: Setting default profile to HD 720p 30 fps 15:35:30 INFO preview_thread: player Position(): 1 15:35:30 INFO video_widget: Load: Set video widget display aspect ratio to: 1.7777777910232544 15:35:30 INFO video_widget: Set video widget pixel aspect ratio to: 1.0 15:35:30 INFO main_window: updateStatusChanged 15:35:30 INFO main_window: foundCurrentVersion: Found the latest version: 2.5.1 15:35:31 INFO preview_thread: player Position(): 1 15:35:31 INFO webengine: Registering WebChannel connection with WebEngine 15:35:31 INFO timeline: Adjusting max size of preview image: PyQt5.QtCore.QSize(718, 404) 15:35:31 INFO preview_thread: player Position(): 1 15:35:31 ERROR webengine: angular-animate.min.js@L14: Uncaught TypeError: m.module(...).info is not a function 15:35:31 INFO webview: Qt Ready 15:35:31 INFO webview: Angular Ready 15:35:44 INFO main_window: Clear all thumbnails: /home/navin/.openshot_qt/thumbnail 15:35:44 INFO main_window: Clear all animations: /home/navin/.openshot_qt/blender 15:35:44 INFO main_window: Clear all titles: /home/navin/.openshot_qt/title 15:35:44 INFO project_data: Setting default profile to HD 720p 30 fps 15:35:44 INFO project_data: Loading project file: /home/navin/Documents/ARG/OpenSHot Files/Untitled Project1.osp 15:35:44 INFO assets: Using existing asset folder /home/navin/Documents/ARG/OpenSHot Files/Untitled Project1_assets 15:35:44 INFO assets: Using existing asset folder /home/navin/Documents/ARG/OpenSHot Files/Untitled Project1_assets 15:35:44 INFO assets: Using existing asset folder /home/navin/Documents/ARG/OpenSHot Files/Untitled Project1_assets 15:35:44 INFO project_data: checking project files... 15:35:44 INFO project_data: checking file /home/navin/Desktop/rvgtrailer5.mp4 15:35:44 INFO project_data: checking file /home/navin/Desktop/untitled1.mp3 15:35:44 INFO project_data: checking file /home/navin/Documents/ARG/EPISODES/13/Untitled Project1.mp3 15:35:44 INFO project_data: checking file /home/navin/Desktop/untitled.mp3 15:35:44 INFO project_data: checking file /home/navin/Downloads/boy-2328038_640.png 15:35:44 INFO project_data: checking file /home/navin/Downloads/On-air-Thinkstocck.jpg 15:35:44 INFO project_data: checking file /home/navin/Downloads/american-christmas-traditions-gettyimages-487756624.jpg 15:35:44 INFO project_data: checking file /home/navin/Downloads/oekaki-2009817_640.jpg 15:35:44 INFO project_data: checking file /home/navin/Documents/ARG/EPISODES/13/title.png 15:35:44 INFO project_data: checking file /home/navin/Documents/ARG/EPISODES/13/DISWELL.png 15:35:44 INFO project_data: checking file /home/navin/Downloads/index999999999999.jpeg 15:35:44 INFO project_data: checking file /home/navin/Downloads/Waipod Phetsuphan , Ding Ding Dong.mp4.mp3 15:35:44 INFO project_data: checking file /home/navin/Downloads/zen-garden.mp3 15:35:44 INFO project_data: checking file /home/navin/Downloads/underglow.mp3 15:35:44 INFO project_data: checking file /home/navin/Documents/Projects/YouTube/RVG/Every Episode/deleted scenes.mp4 15:35:44 INFO project_data: checking file /home/navin/Documents/Projects/YouTube/RVG/Every Episode/y2mate.com - vietnam_traditional_folk_music_yvoJdhkxYIs_160kbps.webm 15:35:44 INFO project_data: checking file /home/navin/Documents/Projects/YouTube/RVG/Misc/endtitle.png 15:35:44 INFO project_data: checking file /home/navin/Desktop/90sweb.png 15:35:44 INFO project_data: checking file /home/navin/Downloads/metaphor-1209691_1920.jpg 15:35:44 INFO project_data: checking file /home/navin/Downloads/33722ca883044584c5ce7e1a8ee96550.webp 15:35:44 INFO project_data: checking file /home/navin/Downloads/error-2129569_1280.jpg 15:35:44 INFO project_data: checking file /home/navin/Desktop/trailerwallpaper.jpg 15:35:44 INFO project_data: checking file /home/navin/Desktop/bloopers.png 15:35:44 INFO project_data: checking file /home/navin/Downloads/179223__efecto-fundador__digital-error.wav 15:35:44 INFO project_data: checking file /home/navin/Desktop/kradenoe_solntse_1943_512kb.mp4 15:35:44 INFO project_data: checking file /home/navin/Desktop/pagan-cross.mp3 15:35:44 INFO project_data: checking file /home/navin/Downloads/al-andalus.mp3 15:35:44 INFO project_data: checking file /home/navin/Desktop/personalanswer.png 15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME10-RVG.png 15:35:44 INFO project_data: checking file /home/navin/Downloads/pagan-cross.mp3 15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME9-RVG.png 15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME8-RVG.png 15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME7-RVG.png 15:35:44 INFO project_data: checking file /home/navin/Downloads/dire-space-emergency.mp3 15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME6-RVG.png 15:35:44 INFO project_data: checking file /home/navin/Downloads/love-3556651_640.png 15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME5-RVG.png 15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME4-RVG.png 15:35:44 INFO project_data: checking file /home/navin/Downloads/signs-to-nowhere.mp3 15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME3-RVG.png 15:35:44 INFO project_data: checking file /home/navin/Downloads/index99.jpeg15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME2-RVG.png 15:35:44 INFO project_data: checking file /home/navin/Desktop/fix.png 15:35:44 INFO project_data: checking file /home/navin/Desktop/COMMENT-NAME-RVG.png 15:35:44 INFO project_data: checking file /home/navin/Downloads/subtitle video.mp4 15:35:44 INFO project_data: checking file /home/navin/Desktop/wall-1217083_1280(1).jpg 15:35:44 INFO project_data: checking file /home/navin/Downloads/index.jpeg 15:35:44 INFO project_data: checking file /home/navin/Downloads/insfdfsdfdex.jpeg 15:35:44 INFO project_data: checking file /home/navin/Downloads/Lilac-Bowls-335ml-Plastic-LAVE2BOWL_th2.jpg 15:35:44 INFO project_data: checking file /home/navin/Downloads/cheesy-ranch-burger.jpg 15:35:44 INFO project_data: checking file /home/navin/Downloads/images.jpeg 15:35:44 INFO project_data: checking file /home/navin/Downloads/indekkkx.jpeg 15:35:44 INFO project_data: checking file /home/navin/Downloads/index890.jpeg 15:35:44 INFO project_data: checking file /home/navin/Downloads/index98.jpeg 15:35:44 INFO project_data: checking file /home/navin/Downloads/index9.jpeg 15:35:44 INFO project_data: checking file /home/navin/Downloads/710tVLiJOsL.jpg 15:35:44 INFO project_data: checking file /home/navin/Downloads/HLB1c9ehP9rqK1RjSZK9q6xyypXaD.jpg 15:35:44 INFO project_data: checking file /home/navin/Downloads/food-2202399_640.jpg 15:35:44 INFO project_data: checking file /home/navin/Downloads/boy-179376_640.jpg 15:35:44 INFO project_data: checking file /home/navin/Downloads/worm-1439069_640.jpg 15:35:44 INFO project_data: checking file /home/navin/Downloads/ca63640d6a175b2b259ad0396b094c19.jpg 15:35:44 INFO project_data: checking file /home/navin/Downloads/napkins-5073819_640.jpg 15:35:44 INFO project_data: checking file /home/navin/Downloads/il_340x270.774243765_5c7h.jpg 15:35:44 INFO project_data: checking file /home/navin/Downloads/fancy-a-spit-roast-card.jpg 15:35:44 INFO project_data: checking file /home/navin/Downloads/Filthy-Animal-Greeting-Card.webp 15:35:44 INFO project_data: checking file /home/navin/Desktop/Rvg-backgroundmusic-trailer.wav 15:35:44 INFO project_data: checking file /home/navin/Desktop/rvgtrailer4.mp4 15:35:44 INFO project_data: checking file /home/navin/Downloads/rvg 13.mp4 15:35:44 INFO project_data: Project data: openshot 2.5.1-dev2, libopenshot 0.2.5-dev2 15:35:45 INFO preview_thread: player Position(): 1 15:35:46 INFO video_widget: Load: Set video widget display aspect ratio to: 1.7777777910232544 15:35:46 INFO video_widget: Set video widget pixel aspect ratio to: 1.0 15:35:46 INFO main_window: updateStatusChanged 15:35:46 INFO main_window: updateStatusChanged 15:35:46 INFO main_window: updateStatusChanged 15:35:46 INFO main_window: updateStatusChanged 15:35:46 INFO preview_thread: player Position(): 1 15:35:46 INFO main_window: Loaded project /home/navin/Documents/ARG/OpenSHot Files/Untitled Project1.osp 15:35:51 INFO preview_thread: previewFrame: 458, player Position(): 458`

AND THE OTHER ERROR LOGFILE

`libopenshot logging: Wed Dec 2 15:35:28 2020

---- Unhandled Exception: Stack Trace ---- /lib/x86_64-linux-gnu/libopenshot.so.19 openshot::FFmpegReader::Close() 0x62c [0x7f84f813091c] /lib/x86_64-linux-gnu/libopenshot.so.19 openshot::FrameMapper::Close() 0x269 [0x7f84f8156989] /lib/x86_64-linux-gnu/libopenshot.so.19 openshot::Clip::Close() 0x26e [0x7f84f810f23e] /lib/x86_64-linux-gnu/libopenshot.so.19 openshot::Timeline::update_open_clips(openshot::Clip*, bool) 0x6b5 [0x7f84f816ccc5] /lib/x86_64-linux-gnu/libopenshot.so.19 openshot::Timeline::find_intersecting_clips(long, int, bool) 0x36a [0x7f84f816d79a] /lib/x86_64-linux-gnu/libopenshot.so.19 openshot::Timeline::GetFrame(long) 0x7b1 [0x7f84f816e151] /lib/x86_64-linux-gnu/libopenshot.so.19 openshot::PlayerPrivate::getFrame() 0x7f [0x7f84f8195fff] /lib/x86_64-linux-gnu/libopenshot.so.19 openshot::PlayerPrivate::run() 0xa1 [0x7f84f8196121] /lib/x86_64-linux-gnu/libopenshot-audio.so.7 juce::Thread::threadEntryPoint() 0xd9 [0x7f84e513eb09] /lib/x86_64-linux-gnu/libopenshot-audio.so.7 0x136c3d [0x7f84e513ec3d] /lib/x86_64-linux-gnu/libpthread.so.0 0x9609 [0x7f851780c609] /lib/x86_64-linux-gnu/libc.so.6 clone 0x43 [0x7f8517948293] ---- End of Stack Trace ---- ---- Unhandled Exception: Stack Trace ----`

**Screenshots:** 
Not applicable

**Comment by USATechDude** [comment]
@Innitinnuit Would you attach the log files instead of Cut & Paste them into the body, please?  It is easier for some of us to read the log files in their native format.

Did you delete your ~/.openshot_qt folder then relaunch OpenShot?  Sorry if I am asking if you did this when you already have, but I am trying to establish a baseline for my reference.

**Comment by Innitinnuit** [comment]
@USATechDude   
No need for apologies. You're like a God to me and I'm humble. Yes I did delete the mentioned folder and relaunch - sorry for not mentioning that. I've followed every piece of advice I could find before coming here as a last resort and bothering people.
Files attached from the debugging. Because I deleted the folder previously I don't have the non-debugging logs, but let me know if you want me to recreate again.

[openshot-qt.log](https://github.com/OpenShot/openshot-qt/files/5639030/openshot-qt.log)
[libopenshot.log](https://github.com/OpenShot/openshot-qt/files/5639032/libopenshot.log)

**Comment by Innitinnuit** [comment]
@USATechDude 
I tried with the stable and daily build version. Also I downloaded the appimage and ran it as an executable file - both for the stable and daily build versions.

**Comment by USATechDude** [comment]
When you purged your PPA version before you tried the stable PPA, did you also remove the folder 
`/usr/lib/python3/dist-packages/openshot_qt`?

**Comment by USATechDude** [comment]
@Innitinnuit, I'm just writing up a short note to let anyone who reads this know that we've resolved the issue that OpenShot was crashing.

I asked Innitinnuit to purge the instance of OpenShot from his Ubuntu 20.04 system using:
`sudo apt-get purge openshot-qt`

After purging OpenShot, I asked him to remove the openshot-qt folder located at:
`/usr/lib/python3/dist-packages/openshot-qt`

I also asked him to remove the PPA repository pointing to the stable version using:
`sudo add-apt-repository -r ppa:openshot.developers/ppa`

He then followed the instructions for adding the Daily Build repository and installing OpenShot.
`sudo add-apt-repository ppa:openshot.developers/libopenshot-daily`
`sudo apt-get update`
`sudo apt-get install openshot-qt`

After reviewing the logs, I asked Innitinnuit to reinstall `python3` and install `python-is-python3`.  We later discovered that his system did not have `pip`.  After installing `pip`, he reinstalled `PyQt5`.

**Comment by Innitinnuit** [comment]
Thanks so much @USATechDude  for the 6 hours of your free time you spent helping me with this. You are amazing and so is openshot. I'll definitely be making a donation!

**Comment by Innitinnuit** [comment]
This issue wasn't technically resolved. 

I appreciate techdude spending 6 hours on the phone with me, truly exceptional work. He was more professional than any paid tech staff I have ever encountered - who usually get angry after 20 mins. His knowledge was astounding and he was able to think of a multitude of different solutions for this. I truly have great admiration for openshot and what they do. 

However, we ended the call without being able to fix the issue and also afterwards my linux installation became a weird frankenstein: the ubuntu DE began mixing with elements of LXqt. For example, the gnome terminal within ubuntu became LXqt as well as a number of other issues. It has taken hours and numerous questions on Ask Ubuntu to fix these. The creator of Openshot was contacted and switftly replied - amazing again- but I feel it's slightly disingenuous to claim this issue was resolved. I'm still having issues with new projects crashing after a certain amount of time.
